### DISCUSSION TITLE
Is there a way to export the current config?

### DISCUSSION THREAD

**Discussion by jottr** [ianyh/Amethyst#1432] [Q&A]
I'd like to manage my configuration in a configuration file.  
I've previous configured Amethyst from the GUI settings panel. Is there a way to export my current config to a file, or does it exist somewhere  on the filesystem?

Also, does Amethyst honor the MacOS specific config directory convention, i.e. `~/Library/Application Support/Amethyst`?  
Or, can the config dir be set to an arbitrary location as per the users needs (e.g. `~/.config/amethyst`)?

**Comment by ianyh** [comment]
See this for configuration file details, but there is not currently a way to export configuration from the interface to a file.

**Reply by jottr** [reply]
I suspect you meant to attach a link to `see this`?

**Reply by ianyh** [reply]
🤦 

https://github.com/ianyh/Amethyst/blob/development/docs/configuration-files.md

**Reply by jottr** [reply]
Yeah, I'm aware of that document. :)

**Reply by jottr** [reply]
What about the config directory location? Which directories will be picked up by Amethyst?

**Reply by jottr** [reply]
Ah, I see. 
 https://github.com/ianyh/Amethyst/blob/c48f35a8421c7c11f116b9fb804ef9281d9f908a/Amethyst/Preferences/UserConfiguration.swift#L330-L335

It looks like one would have to add [`applicationSupportDirectory`](https://developer.apple.com/documentation/foundation/url/3988452-applicationsupportdirectory/) to the search path, if one would want to honor MacOS conventions, correct?
